Version.xcconfig

```diff
@@ -6,9 +6,6 @@
 //  Copyright © 2020 LoopKit Authors. All rights reserved.
 //
 
-// Version [DEFAULT]
-LOOP_MARKETING_VERSION = 3.5.0
-CURRENT_PROJECT_VERSION = 57
-
-// Optional override
-#include? "VersionOverride.xcconfig"
```

Comment:
I believe if you add a VersionOverride.xcconfig to the workspace, you can override it there; that will remain compatible with the Tidepool build system.

Reply:
I added on line to enable looking for VersionOverride.xcconfig file in LoopWorkspace